src/applications/claims-status/tests/components/TimezoneDiscrepancyMessage.unit.spec.jsx

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import sinon from 'sinon';
+
+import TimezoneDiscrepancyMessage from '../../components/TimezoneDiscrepancyMessage';
+
+// Helper function to create store with dynamic toggle control
+const getStore = (toggleEnabled = true) =>
+  createStore(() => ({
+    featureToggles: {
+      // eslint-disable-next-line camelcase
+      cst_timezone_discrepancy_mitigation: toggleEnabled,
+    },
+  }));
+
+describe('<TimezoneDiscrepancyMessage>', () => {
+  let timezoneStub;
+
+  beforeEach(() => {
+    // Stub CST timezone (-360 = UTC-6) for all tests by default
+    // Individual tests can override by restoring and re-stubbing
+    timezoneStub = sinon
+      .stub(Date.prototype, 'getTimezoneOffset')
+      .returns(-360);
+  });
+
+  afterEach(() => {
+    if (timezoneStub) {
+      timezoneStub.restore();
+      timezoneStub = null;
+    }
+  });
+
+  it('should render message with correct text', () => {
+    const { getByText } = render(
+      <Provider store={getStore(true)}>
+        <TimezoneDiscrepancyMessage />
+      </Provider>,
+    );
+
+    expect(getByText(/Files uploaded (after|before).*will show (with|as)/)).to
+      .exist;
+  });
+
+  it('should display time and timezone abbreviation in message', () => {
+    const { getByText } = render(
+      <Provider store={getStore(true)}>
+        <TimezoneDiscrepancyMessage />
+      </Provider>,
+    );
+
+    const message = getByText(/Files uploaded (after|before)/);
+    // Should include time format (H:MM a.m./p.m.) and timezone abbreviation
+    expect(message.textContent).to.match(/\d{1,2}:\d{2}\s+(a|p)\.m\./);
```

Comment:
I don't think we need this line it appears redundant. The expect block below includes everything that this one does.